workers/common/src/main/kotlin/common/env/ConfigFileBuilder.kt

```diff
@@ -162,3 +216,14 @@ data class ProxyConfig(
     val httpsProxy: String?,
     val noProxy: String?
 )
+
+data class ProxiesFromSystemProperties(
```

Comment:
Public classes should have a comment. Here, it would be especially useful to describe the differences to `ProxyConfig`, since semantically both classes are equivalent. I think, ideally, only a single class would be used to hold the proxy configuration; but I understand that different representations of the data are needed.

BTW, should the class be named `ProxyConfigFromSystemProperties` ?

Reply:
Added comments and renamed the class.